File: PRNumberBiz/app/src/main/java/com/pplus/prnumberbiz/apps/common/toolbar/ImplToolbar.java
package com.pplus.prnumberbiz.apps.common.toolbar;

import androidx.annotation.NonNull;

/**
 * Created by j2n on 2016. 8. 12..
 */
public interface ImplToolbar{

    /**
     * toolbar option 정의함
     * */
    @NonNull
    ToolbarOption getToolbarOption();

    OnToolbarListener getOnToolbarClickListener();
}

File: Buflexz/app/src/main/java/com/root37/buflexz/apps/common/toolbar/ImplToolbar.kt
package com.root37.buflexz.apps.common.toolbar

/**
 * Created by j2n on 2016. 8. 12..
 */
interface ImplToolbar {
    /**
     * toolbar option 정의함
     */
    fun getToolbarOption(): ToolbarOption?
    fun getOnToolbarClickListener(): OnToolbarListener?
}